tutorials/roofit/rf101_basics.py

```diff
@@ -0,0 +1,74 @@
+## \file
+## \ingroup tutorial_roofit
+## \notebook
+## This tutorial illustrates the basic features of RooFit.
+##
+## \macro_code
+##
+## \date February 2018
+## \author Clemens Lange
+## \author Wouter Verkerke (C version)
+
+import ROOT
+
+# Setup model
+# ---------------------
+# Declare variables x,mean,sigma with associated name, title, initial
+# value and allowed range
+x = ROOT.RooRealVar("x", "x", -10, 10)
+mean = ROOT.RooRealVar("mean", "mean of gaussian", 1, -10, 10)
+sigma = ROOT.RooRealVar("sigma", "width of gaussian", 1, 0.1, 10)
+
+# Build gaussian p.d.f in terms of x,mean and sigma
+gauss = ROOT.RooGaussian("gauss", "gaussian PDF", x, mean, sigma)
+
+# Construct plot frame in 'x'
+xframe = x.frame(ROOT.RooFit.Title("Gaussian p.d.f."))  # RooPlot
+
+# Plot model and change parameter values
+# ---------------------------------------------------------------------------
+# Plot gauss in frame (i.e. in x)
+gauss.plotOn(xframe)
+
+# Change the value of sigma to 3
+sigma.setVal(3)
+
+# Plot gauss in frame (i.e. in x) and draw frame on canvas
+gauss.plotOn(xframe, ROOT.RooFit.LineColor(ROOT.kRed))
+
+# Generate events
+# -----------------------------
```

Comment:
These "-------" lines will actually make the previous line appear like a header in a markdown cell, so it is fine (the text will be bigger).